cosmic-core/api/src/main/java/com/cloud/api/response/WhoHasThisIpResponse.java

```diff
@@ -0,0 +1,150 @@
+package com.cloud.api.response;
+
+import com.cloud.api.ApiConstants;
+import com.cloud.api.BaseResponse;
+import com.cloud.network.IpAddress;
+import com.cloud.network.Networks;
+import com.cloud.serializer.Param;
+import com.cloud.utils.net.Ip;
+import com.cloud.vm.Nic;
+
+import java.net.URI;
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+
+public class WhoHasThisIpResponse extends BaseResponse {
```

Comment:
I think it's a good idea to also include the UUID next to the name of the resources that are being listed in the response. Something that could be shown only in the detailed view for example. That way we can avoid people looking up these UUIDs in the database.